samples/WebApi.Shared/ApiModel/CompanyModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace WebApi.ApiModel
+{
+    public class CompanyModel
+    {
+        public string CompanyCode { get; set; }
+        public string CompanyName { get; set; }
```

Comment:
I think we can remove the "Company" prefix because they are inside a CompanyModel